README.md

```diff
@@ -3,25 +3,25 @@
 ## project sructure
 
 - ./client - frontend side
-- ./ - backend side
+- ./server - backend side
 
 ### to run project
 
-1. npm run install-all // it will install dependencies inside root and client folder
-2. npm run dev // it will run at the same time server + react-app
+1. npm run install:all // it will install dependencies inside root and client folder
```

Comment:
This script naming style is recommended? I read once this:
https://stackoverflow.com/questions/45034124/what-naming-convention-do-i-use-for-scripts-in-package-json
and opinions are divided

Reply:
We haveto jump in something and keep it to the end

Reply:
I believe camel case would be the best option. I will another task for that